mathml/relations/html5-tree/css-inline-style-interface.tentative.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>MathML 'ElementCSSInlineStyle` Mixin Tests</title>
+    <link
+      rel="help"
+      href="https://mathml-refresh.github.io/mathml-core/#dom-and-javascript"
+    />
+    <style>
+        math * { background-color: red; }
+    </style>
+    <meta name="assert" content="MathMLElements incorporate a functional ElementCSSInlineStyle interface" />
+    <script src="/mathml/support/mathml-fragments.js"></script>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+
+  </head>
+  <body>
+    <span>This tests the presence and functionality of features of the `ElementCSSInlineStyle` interface for MathMLElements</span>
+    <math>
+      <mi>E</mi>
```

Comment:
the math can be empty if you set its content later

Reply:
this should be resolved.